clients/node/client-rds-data-node/protocol/ExecuteStatement.ts

```diff
@@ -1,5 +1,5 @@
 import { HttpRequest, HttpResponse } from "@aws-sdk/protocol-http";
-import { SerializerUtils, DeserializerUtils } from "@aws-sdk/types";
+import { SerializerUtils, DeserializerUtils, Endpoint } from "@aws-sdk/types";
```

Comment:
nit: unused import?

Reply:
We should enforce no unused imports in the tsconfig. Some packages already do

Reply:
Resolved in https://github.com/aws/aws-sdk-js-v3/pull/404/commits/4387c11f42651b3c5222af1a6fc8ae999e7b58db